docs/source/usage.rst

```diff
@@ -33,6 +33,32 @@ For example:
 
    print(prediction_uuid, ml_output) # Print the returned output
 
+Listing Past Predictions
+-------------------------
+
+To list past predictions, make use of the ``konan_sdk.sdk.KonanSDK.get_predictions()`` method, which
+uses `yield`-based semantics to both allow you to retrieve a large number of predictions while
+also maintaining low request latency and response size.
+
+.. code-block:: python
+
+   import datetime
+   from konan_sdk.sdk import KonanSDK
+
+   sdk = KonanSDK(verbose=False) # Initialize the SDK.
+   user = sdk.login("<email>", "<password>") # Login user your valid konan credentials
```

Comment:
Can you replace this with the api key login or add a separate section for login with api key?

Reply:
Done